Storage/src/StorageObject.php

```diff
@@ -935,14 +980,18 @@ public function signedUploadUrl($expires, array $options = [])
      *
      *     @type string $contentType If you provide this value, the client must
      *           provide this HTTP header set to the same value.
+     *     @type string $origin Value of CORS header
+     *           "Access-Control-Allow-Origin". **Defaults to** "*".
```

Comment:
Small nit, could we use backticks around the asterisk?

Reply:
Thanks for the catch! I did a check of the code, and convention here would be

```
**Defaults to** `"*"`.
```

So I updated to match that format. :)